### DISCUSSION TITLE
Turn off color explicitly

### DISCUSSION THREAD

**Discussion by cshen-dev** [slog-rs/slog#326] [Q&A]
Hi slog community!
I've been using `qdrant` as my vector store and it uses `slog` according to their `Cargo.toml`. However, as a rust newbie, I have no idea how to disable the colors in logging by setting some environment variables such as `no_color=true`.

Is that a lacking feature from `slog-term` or am I missing the document?

**Comment by dpc** [comment]
[I don't think slog-term respects NO_COLOR](https://github.com/search?q=repo%3Aslog-rs%2Fterm%20NO_COLOR&type=code) and it probably should.
